helpers/sendMail.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require("nodemailer");
+// const { HttpError } = require("../helpers");
+const { SERVER_URL, UN_PASS } = process.env;
+
+const sendMail = async (email, verificationToken) => {
+  const config = {
+    host: "smtp.ukr.net",
+    port: 465,
+    secure: true,
+    auth: {
+      user: "goit_nodejs77@ukr.net",
```

Comment:
this should be in .env file. Email is confident info

Reply:
Fixed